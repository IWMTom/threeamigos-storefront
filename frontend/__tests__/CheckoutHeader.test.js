import CheckoutHeader from "../components/CheckoutHeader";
import { shallow, mount } from "enzyme";
import toJSON from "enzyme-to-json";

const step = 3;

describe("<CheckoutHeader />", () => {
	it("renders and matches the snapshot", () => {
		const wrapper = shallow(<CheckoutHeader step={step} />);

		expect(toJSON(wrapper)).toMatchSnapshot();
	});
});
