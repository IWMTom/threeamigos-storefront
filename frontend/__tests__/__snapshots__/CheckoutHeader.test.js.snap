// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CheckoutHeader /> renders and matches the snapshot 1`] = `
<Fragment>
  <div
    className="page-header"
  >
    <div
      className="container"
    >
      <h1>
        <i
          className="ic_shopping_cart"
        />
        <span>
          Checkout
        </span>
      </h1>
    </div>
  </div>
  <div
    id="checkout-steps"
  >
    <div
      className="container"
    >
      <div
        className="pure-g"
      >
        <div
          className="pure-u-1-4"
        >
          <div
            className="checkout-step"
          >
            <i
              className="ic_shopping_cart"
            />
            <div>
              <span>
                Step 1
              </span>
              <p>
                Shopping Cart
              </p>
            </div>
          </div>
        </div>
        <div
          className="pure-u-1-4"
        >
          <div
            className="checkout-step"
          >
            <i
              className="ic_credit-card"
            />
            <div>
              <span>
                Step 2
              </span>
              <p>
                Payment Method
              </p>
            </div>
          </div>
        </div>
        <div
          className="pure-u-1-4"
        >
          <div
            className="checkout-step done"
          >
            <i
              className="ic_truck"
            />
            <div>
              <span>
                Step 3
              </span>
              <p>
                Delivery Address
              </p>
            </div>
          </div>
        </div>
        <div
          className="pure-u-1-4"
        >
          <div
            className="checkout-step"
          >
            <i
              className="ic_checkmark"
            />
            <div>
              <span>
                Step 4
              </span>
              <p>
                Confirmation
              </p>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</Fragment>
`;
